Tests/SwiftProtobufPluginLibraryTests/Test_FeatureResolver.swift

```diff
@@ -61,6 +61,19 @@ final class Test_FeatureResolver: XCTestCase {
         }
     }
 
+    func testInit_EditionUnstable() {
```

Comment:
This needs a `throws` effect since it `try`s inside.

```suggestion
    func testInit_EditionUnstable() throws {
```